lld/test/ELF/arm-thumb-jump8-11.s

```diff
@@ -0,0 +1,32 @@
+# REQUIRES: arm
+
+# RUN: llvm-mc -triple thumbv6m-arm-eabi --filetype=obj %s -o %t.obj
+# RUN: ld.lld %t.obj -o %t.linked
+# RUN: llvm-objdump -d %t.linked | FileCheck %s --check-prefixes=CHECK,CHECK-LE
+
+# RUN: llvm-mc -triple thumbebv6m-arm-eabi --filetype=obj %s -o %t.obj
+# RUN: ld.lld %t.obj -o %t.linked
+# RUN: llvm-objdump -d %t.linked | FileCheck %s --check-prefixes=CHECK,CHECK-BE
+
+# CHECK-LE:                    file format elf32-littlearm
+# CHECK-BE:                    file format elf32-bigarm
+
+# CHECK:                       Disassembly of section .text:
+
+# CHECK-LABEL: [[#%x,TARGET:]] <target>:
+# CHECK-NEXT:      [[#TARGET]]: 4770    bx lr
```

Comment:
Since the address 4770 is not important. Just omit it so that future address layout changes would not need to touch this file.